PairInteraction.jl/src/PairInteraction.jl

```diff
@@ -0,0 +1,34 @@
+module PairInteraction
+
+import Base.^
+import Base.string
+using CxxWrap
+
+pairinteraction_dir = dirname(@__FILE__)
+build_dir = joinpath(splitdir(splitdir(pairinteraction_dir)[1])[1], "build", "pairinteraction")
```

Comment:
I replaced `splitpath` since that is only available in Julia 1.1 but the Docker containers ship 1.0.5 (which I would prefer to keep because it's the LTS variant).